includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1922,6 +1961,427 @@ public static function print_status_meta_box( $post ) {
+				<p>
+					<?php esc_html_e( 'You are nearing the limit of the CSS budget. Once reaching this limit, stylesheets deemed of lesser priority will be excluded from the page. Please review the stylesheets below and determine if the current theme or a particular plugin is including excessive CSS.', 'amp' ); ?>
+				</p>
+			</div>
+		<?php endif; ?>
+
+		<table class="amp-stylesheet-list wp-list-table widefat fixed striped">
+			<thead>
+			<tr>
+				<th class="column-stylesheet_expand"></th>
+				<th class="column-stylesheet_order"><?php esc_html_e( 'Order', 'amp' ); ?></th>
+				<th class="column-original_size"><?php esc_html_e( 'Original Size', 'amp' ); ?></th>
+				<th class="column-minified"><?php esc_html_e( 'Minified', 'amp' ); ?></th>
+				<th class="column-final_size"><?php esc_html_e( 'Final Size', 'amp' ); ?></th>
+				<th class="column-percentage"><?php esc_html_e( 'Percentage', 'amp' ); ?></th>
+				<th class="column-priority"><?php esc_html_e( 'Priority', 'amp' ); ?></th>
+				<th class="column-stylesheet_status"><?php esc_html_e( 'Status', 'amp' ); ?></th>
+				<th class="column-markup"><?php esc_html_e( 'Markup', 'amp' ); ?></th>
+				<th class="column-sources_with_invalid_output"><?php esc_html_e( 'Sources', 'amp' ); ?></th>
+			</tr>
+			</thead>
+			<tbody>
+			<?php $row = 1; ?>
+			<?php foreach ( $stylesheets as $stylesheet ) : ?>
+				<?php
+				// @todo Add information about amp-keyframes as well.
+				if ( ! isset( $stylesheet['group'] ) || 'amp-custom' !== $stylesheet['group'] || ! empty( $stylesheet['duplicate'] ) ) {
+					continue;
+				}
+
+				$origin_html = '<' . $stylesheet['element']['name'];
+				if ( ! empty( $stylesheet['element']['attributes'] ) ) {
+					$attributes = $stylesheet['element']['attributes'];
+					if ( ! empty( $attributes['class'] ) ) {
+						$attributes['class'] = trim( preg_replace( '/(^|\s)amp-wp-\w+(\s|$)/', ' ', $attributes['class'] ) );
+						if ( empty( $attributes['class'] ) ) {
+							unset( $attributes['class'] );
+						}
+					}
+					if ( isset( $attributes[ AMP_Style_Sanitizer::ORIGINAL_STYLE_ATTRIBUTE_NAME ] ) ) {
+						$attributes['style'] = $attributes[ AMP_Style_Sanitizer::ORIGINAL_STYLE_ATTRIBUTE_NAME ];
+						unset( $attributes[ AMP_Style_Sanitizer::ORIGINAL_STYLE_ATTRIBUTE_NAME ] );
+					}
+					if ( ! empty( $attributes ) ) {
+						$origin_html .= ' ' . AMP_HTML_Utils::build_attributes_string( $attributes );
+					}
+				}
+				$origin_html .= '>';
+
+				$ratio = $stylesheet['final_size'] / $stylesheet['original_size'];
+				?>
+				<tr class="<?php echo esc_attr( sprintf( 'stylesheet level-0 %s', 0 === $row % 2 ? 'even' : 'odd' ) ); ?>">
+					<td class="column-stylesheet_expand">
+						<button class="toggle-stylesheet-details" type="button">
+							<span class="screen-reader-text"><?php esc_html_e( 'Expand/collapse', 'amp' ); ?></span>
+						</button>
+					</td>
+					<td class="column-stylesheet_order">
+						<?php echo (int) $row; ?>
```

Comment:
`$row` is already an `int` here.

Reply:
Yes. This is done purely as a way to explicitly mark the variable as safe for printing. So it's an alternative to doing `esc_html()`.